registry/consul/parse_test.go

```diff
@@ -47,6 +47,12 @@ func TestParseTag(t *testing.T) {
 			route: "xx/Yy",
 			ok:    true,
 		},
+		{
+			tag:   "p-www.bar.com:80/foo https://www.bar.com/ redirect=302",
+			route: "www.bar.com:80/foo",
+			opts:  "https://www.bar.com/ redirect=302",
```

Comment:
Ah, I forgot that we need to be able to specify the redirect target. How about this?

```
redirect=302,http://www.bar.com/
```

Reply:
Sure I can do that syntax for the consul service tags.